app-frontend/packs/components/dashboard/homePage/HomePageView.vue

```diff
@@ -1,19 +1,54 @@
 <template>
 
-    <LayoutPage>
-        <HomePage></HomePage>
-    </LayoutPage>
+    <div v-if="!isLoader">
+
+        <HomePageEmptyState
+            @create="createBanner"
+            v-if="isLengthData"
+        />
+
+        <HomePageWithBanners
+            @create="createBanner"
+            v-else
+        />
+
+    </div>
+
+    <LoaderPage v-if="isLoader"/>
 
 </template>
 
 <script>
 
-import HomePage from './HomePage';
+import HomePageEmptyState from "./HomePageEmptyState";
+import HomePageWithBanners from "./HomePageWithBanners";
+import router from "../../../router/router";
+import LoaderPage from "../../baseComponents/LoaderPage";
+import {mapActions, mapState} from 'vuex'
 
 export default {
     name: 'HomePageView',
     components: {
-        HomePage
+        LoaderPage,
+        HomePageWithBanners,
+        HomePageEmptyState,
+    },
+    methods: {
+        createBanner() {
+            router.push('/editor');
+        },
+        ...mapActions({
+            fetchBanners: 'banners/fetchBanners',
+        })
+    },
+    computed: {
+        ...mapState ({
+            isLoader: state => state.banners.isLoader,
+            isLengthData: state => state.banners.isLengthData
```

Comment:
не очень понятное название

лучше уже так
```suggestion
            isEmpty: state => state.banners.isEmpty
```

Reply:
добре